src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -285,16 +285,20 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
             $listeners[] = new Reference($this->createContextListener($container, $contextKey));
         }
 
+        // Determine default entry point
+        $configuredEntryPoint = isset($firewall['entry_point']) ? $firewall['entry_point'] : null;
```

Comment:
should probably be moved back as this is not needed

Reply:
Done